README.md

```diff
@@ -29,24 +29,24 @@ render {
 }.mount("target")
 ```
 
-fritz2 implements **precise data binding**. That means that exactly those (and **only** those) dom-nodes (elements, attributes, etc.) change, that depend on the parts of your data-model, that have changed. 
-There is no intermediate layer needed like a virtual DOM and you do not have to implement any additional methods to decide, which parts of your component have to be rerendered, when your data changes:
+fritz2 implements **precise data binding**. This means that when parts of your data model change, only the DOM-nodes depending on those parts will change as well.
```

Comment:
I think we need to emphasize this more, because this is one of the key benefits of fritz2 compared to react....
Those parts of the DOM (elements or attributes), exactly those and only those... 😇

Reply:
I see what you mean, but that is pretty much already the meaning of "only the dom-nodes depending..". How about we change it to 
```suggestion
The key benefit when using fritz2 is its **precise data binding**. This means that when parts of your data model change, **only** the DOM-nodes depending on them will change as well (and no others).
```

Reply:
This means, that fritz2 automatically updates the DOM-nodes, all the DOM-nodes and nothing but the DOM-nodes depending on parts of your model that change.